src/Umbraco.Web.UI.Client/src/packages/documents/documents/collection/document-collection.context.ts

```diff
@@ -14,13 +32,17 @@ export class UmbDocumentCollectionContext extends UmbDefaultCollectionContext<
 	#displayCulture = new UmbStringState(undefined);
 	#displayCultureObservable = this.#displayCulture.asObservable();
 
+	#propertyValuePresentationManifests: ManifestPropertyValuePresentation[] = [];
+
 	constructor(host: UmbControllerHost) {
 		super(host, UMB_DOCUMENT_TABLE_COLLECTION_VIEW_ALIAS);
 
 		this.consumeContext(UMB_VARIANT_CONTEXT, async (variantContext) => {
 			this.#variantContext = variantContext;
 			this.#observeDisplayCulture();
 		});
+
+		this.#propertyValuePresentationManifests = umbExtensionsRegistry.getByType('propertyValuePresentation');
```

Comment:
I think this should be observing the extenion registry(using byType observer)

The case I'm concerned about is when the user navigates directly to a Collection, in this case some Extentions may not be present yet at this point.

I understand that getting extention manifest updates would at this point not update the rendering. But the rendering still happens a bit later, so that feels more safe to me.

:-)

Reply:
See note below that this could properly be removed.